waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -1463,4 +1463,36 @@ public void update_master_key(int seq_number, String key) throws NoSuchObjectExc
     getPrimaryClient().update_master_key(seq_number, key);
   }
 
```

Comment:
It looks like they've just added some new methods so this should be backwards compatible?

Reply:
as per my previous comment